src/content/docs/products/splashkit/SplashKit Tutorials/Documentation/3-Tutorial-Reviews.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Tutorial Reviews
```

Comment:
I like the intent of this as a means of tracking the reviews that have been done, however I think this might be a good topic to bring up for discussion in a meeting as something about it feels off in terms of tracking work. This is more like BAU work that would be tracked in Planner rather than on the website.

Reply:
This was discussed and organised with Olivia. A planner card has been made for every tutorial that currently exists on the Splashkit website. Each tutorial will be reviewed, and once reviewed the information will be added to the page.


![image](https://github.com/user-attachments/assets/cbcb9c49-2f85-4702-b17c-4215aba5f44c)

Reply:
I am requesting a review of this process at the next standup as seeing it on this site in its current form looks laborious and unintuitive. It's adding way too many steps and a huge amount of administrative overhead for something that Planner is already managing.